platform/viewer/pluginConfig.json

```diff
@@ -25,9 +25,10 @@
       "version": "3.0.1"
     }
   ],
-  "modes": [
+  "modes": [],
```

Comment:
I will revert this entire file before merging, but I wanted to show how it gets used.